Add vacation best practices (#19115)
fleetdm/fleet: handbook/sales/README.md

 - Do not assign the issue. The Director of Solutions Consulting will assign the issue.
 - Ensure that the product category is defined ("Endpoint ops", "Device management", or "Vulnerability management") in the description of the issue.

> ### Hand off technical evaluations in advance of vacation time

Reviewer: ```suggestion
### Hand off a technical evaluation to a temporary DRI
```

> At Fleet, we value vacation time. Fleeties should be able to disconnect completely without having to check email or slack while away from work. In advance of vacation time (target one week in advance), refer to the following examples and review with each individual that will act as the temporary DRI for the technical evaluation while you are away. This can be documented as a google doc or can be added to the relevant tech eval epic issue in github.

Reviewer: Make this less about vacation and more about the handoff. This can be used in any scenario where a handoff is necessary.

Author: copy that.

Ensure that our valued customers know that you will be away and that the temporary DRI has been debriefed on their setup and can handle any technical questions that come up. 

> ```

Reviewer: Add an extra line between your backtics and the text above. I've got PTSD from weird things happening when code blocks are smashed against text.

Author: copy that.
